File: app/src/main/java/com/stratone/bmotion/model/PurchaseHistory.java
package com.stratone.bmotion.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PurchaseHistory {
    @SerializedName("NIP")
    @Expose
    private Object nIP;
    @SerializedName("OrderNo")
    @Expose
    private String orderNo;
    @SerializedName("TransactionDate")
    @Expose
    private String transactionDate;
    @SerializedName("OutletNo")
    @Expose
    private String outletNo;
    @SerializedName("Liter")
    @Expose
    private String liter;

    public Object getNIP() {
        return nIP;
    }

    public void setNIP(Object nIP) {
        this.nIP = nIP;
    }

    public String getOrderNo() {
        return orderNo;
    }

    public void setOrderNo(String orderNo) {
        this.orderNo = orderNo;
    }

    public String getTransactionDate() {
        return transactionDate;
    }

    public void setTransactionDate(String transactionDate) {
        this.transactionDate = transactionDate;
    }

    public String getOutletNo() {
        return outletNo;
    }

    public void setOutletNo(String outletNo) {
        this.outletNo = outletNo;
    }

    public String getLiter() {
        return liter;
    }

    public void setLiter(String liter) {
        this.liter = liter;
    }
}

File: BMotionServices/Models/PurchaseHistory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BMotionServices.Models
{
    public class PurchaseHistory
    {
        string nip;
        public String NIP
        {
            get { return nip; }
            set { nip = value; }
        }

        string orderNo;
        public String OrderNo
        {
            get { return orderNo; }
            set { orderNo = value; }
        }
        string transactionDate;
        public String TransactionDate
        {
            get { return transactionDate; }
            set { transactionDate = value; }
        }

        string outletNo;
        public String OutletNo
        {
            get { return outletNo; }
            set { outletNo = value; }
        }

        string liter;
        public String Liter
        {
            get { return liter; }
            set { liter = value; }
        }
    }
}